src/Sarif.Viewer.VisualStudio/Tags/ISarifLocationTaggerService.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+namespace Microsoft.Sarif.Viewer.Tags
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    [Guid("774C62D3-203A-493C-8801-BEA59FE46CA1")]
+    internal interface ISarifLocationTaggerService
+    {
+        /// <summary>
+        /// Causes a tags changed notification to be sent out from all known taggers.
+        /// </summary>
+        /// <remarks>
+        /// As an example, this method is called is to send a tags changed notification when a "text view" is already open and visible
```

Comment:
>is to [](start = 49, length = 5)

"is to" => "to"

Reply:
still there.

---
In reply to: [502556628](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r502556628) [](ancestors = 502556628)